lib/optimize/SplitChunksPlugin.js

```diff
@@ -101,24 +213,38 @@ const compareEntries = (a, b) => {
 	return compareModuleIterables(modulesB, modulesA);
 };
 
+/**
+ * @param {number} a first number
+ * @param {number} b second number
+ * @returns {number} the difference
+ */
 const compareNumbers = (a, b) => a - b;
 
 const INITIAL_CHUNK_FILTER = chunk => chunk.canBeInitial();
 const ASYNC_CHUNK_FILTER = chunk => !chunk.canBeInitial();
 const ALL_CHUNK_FILTER = chunk => true;
 
+/**
+ * @param {OptimizationSplitChunksSizes} value the sizes
+ * @returns {SplitChunksSizes} normalized representation
+ */
 const normalizeSizes = value => {
 	if (typeof value === "number") {
 		const obj = Object.create(null);
 		obj.javascript = value;
 		return obj;
-	} else if (value && typeof value === "object") {
+	} else if (typeof value === "object" && value !== null) {
 		return Object.assign(Object.create(null), value);
 	} else {
 		return Object.create(null);
 	}
 };
 
+/**
+ * @param {OptimizationSplitChunksSizes} value the sizes
+ * @param {OptimizationSplitChunksSizes} defaultValue the default sizes
+ * @returns {SplitChunksSizes} the merged sizes
+ */
 const mergeSizes = (value, defaultValue) => {
```

Comment:
Extend this to allow more values: `(...values)`

All values should be normalizes and undefined values should be skipped. After that they are fed into `Object.assign` in reverse order.

Reply:
We could reverse the order to avoid reversing the values...